static/js/components/DBStats.jsx

```diff
@@ -41,9 +42,25 @@ const DBStats = () => {
                 <em>{key}</em>
               </TableCell>
               <TableCell>
-                {!Number.isNaN(Number(dbStats[key]))
-                  ? Number(dbStats[key]).toLocaleString()
-                  : dbStats[key]}
+                {/* eslint-disable-next-line no-nested-ternary */}
+                {Array.isArray(dbStats[key]) ? (
+                  <ul style={{ paddingLeft: "0.8rem" }}>
+                    {dbStats[key].map((item) => (
+                      <li key={item.summary}>
+                        {item.summary}
+                        {item.output && (
+                          <Tooltip title={item.output}>
+                            <em> (hover to see output)</em>
```

Comment:
I wonder if the tooltip works for displaying output.  Do we have control over the font in there?  If so, then if we can add appropriate linebreaks and ensure the font is monospace, that would help.  If we can't do that with Tooltip, it may be best to make the entry clickable and fold out the output.

Reply:
Done: 
![proc_output_popover](https://user-images.githubusercontent.com/7230285/108887013-86c18e80-75be-11eb-93e3-1439c6b0909e.png)